.github/pull_request_template.md

```diff
@@ -5,6 +5,7 @@ Please include the following:
 - A meaningful PR title [guidelines](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#use-imperative-mood-in-your-commit-message-subject)
 - A summary of the change
 - Which issue is fixed (if there is one)
+  - Link to the Github issue or Jira ticket
 - Relevant motivation and context
 
 ## How can this be tested?
```

Comment:
Should we also remove the Checklist part? Is someone actually caring about it?

Reply:
> We could shorten the current message/checkboxes, and move most of the things into the CONTRIBUTION.md and only have 1 checkbox that say "[ ] I followed the contribution guidelines". (I would actually prefer this)

I like this! It's just a pain whenever I create a PR without updating tests having to manually strikethrough the checkbox and so